assets/src/edit-story/utils/generatePatternStyles.js

```diff
@@ -117,6 +121,10 @@ function generatePatternStyles(pattern = null, property = 'background') {
     );
   }
 
+  if (pattern === MULTIPLE_VALUE) {
+    return '';
+  }
```

Comment:
Hm, I don't feel like this belongs here.

`MULTIPLE_VALUE` is in implementation detail of the design panel and shouldn't exist here in a general-purpose util function.

Maybe the first abort in `if (pattern ===  null)` can be changed to `if (typeof pattern !== 'object')`. That should also solve this case.

Reply:
So the problem with that is `{ [property]: 'transparent' }` as the return value for `null` also crashes it. The panel really wants an empty string `''`. Alternatively we can update the color picker to handle `MULTIPLE_VALUE` if you want.